src/components/UserAvatar.js

```diff
@@ -7,15 +7,17 @@ import EStyleSheet from 'react-native-extended-stylesheet'
 
 class UserAvatar extends Component {
   render() {
-
     const guestUserLogo =
       <Image source={require('../../images/simple-avatar.png')} style={styles.avatar} />
 
-    const avatar = ( <Image style={styles.avatar} source={
-        this.props.avatar
-        ? {uri: this.props.avatar}
-        : require('../../images/simple-avatar.png') } /> )
-
+    const avatar = (
+      <Image
+        defaultSource={require('../../images/simple-avatar.png')}
```

Comment:
That's in case there is a source that exists (the src is stored in local storage) so it passes the ternary, but we're unable to actually download it because of slow or no internet.  This is an RN helper with Image.